redis/src/main/scala/zio/redis/RedisUri.scala

```diff
@@ -16,8 +16,8 @@
 
 package zio.redis
 
-final case class RedisConfig(host: String, port: Int)
+final case class RedisUri(host: String, port: Int)
```

Comment:
This is not related to the change you're proposing, please revert it.

Reply:
Can we use that name? 😇  I'am going to rename it in cluster PR anyway 😇

Reply:
Have reverted

Reply:
I would still leave it :).